media-gfx/openvdb/files/openvdb-11.0.0-constexpr-version.patch

```diff
@@ -0,0 +1,51 @@
+--- a/nanovdb/nanovdb/NanoVDB.h	2023-11-23 15:03:52.227292525 +0100
```

Comment:
Add comment explaining why this patch is needed in the patch file

Reply:
This makes `Version` a `constexpr` as the rest of the header already is. So we can use it e.g in blender.